docs/src/constructors.md

```diff
@@ -6,14 +6,16 @@ In Julia, one constructs objects of a given type by calling a type constructor.
 with the same name as the type itself. For example, to construct a `BigInt` object in Julia, we simply
 call the `BigInt` constructor:
 
-```
-n = BigInt("1234567898765434567898765434567876543456787654567890")
+```jldoctest
+julia> n = parse(BigInt, "1234567898765434567898765434567876543456787654567890")
```

Comment:
this has been broken all along. the new one doesn't quite fit the text above but I think it is better than something broken